ChangeLog.md

```diff
@@ -2,6 +2,21 @@
 # DGtal 0.9.1
 
 ## New Features / Critical Changes
+- *Base Package*
+ - Traits class for containers in order to probe their category at compile time.
+   (Jacques-Olivier Lachaud, [#1079](https://github.com/DGtal-team/DGtal/pull/1079))
+ - Generic set operations for arbitrary containers. You may use overloaded operators
+   like &, |, -, ^ on arbitrary containers (list, vector, unordered_set, map, etc).
+   (Jacques-Olivier Lachaud, [#1079](https://github.com/DGtal-team/DGtal/pull/1079))
+
```

Comment:
I must check but I don't think it is so easy to do it. I think we will have constructor problems (no default constructor since we need a domain to create a set). Perhaps it is possible with the DigitalSetByAssociativeContainer, by using the copy constructor (sorted specialization does not use default constructor).